puppet/modules/jenkins_job_builder/files/centos.org/ansible/jenkins_job.yml

```diff
@@ -101,6 +103,20 @@
       retries: 360
       delay: 30
 
+    - name: "Create artifacts folder"
+      file:
+        path: "{{ jenkins_artifacts_folder }"
+        state: directory
+      when: jenkins_download_artifacts | bool
+
+    - name: "Fetch remote artifacts"
+      get_url:
+        url: "{{ jenkins_host }}/job/{{ jenkins_job_name }}/{{ poll_result.json.executable.number }}/artifact/{{ item.fileName }}"
```

Comment:
There's no URL in the JSON? :( Should there be a protocol in the URL as well? The rest doesn't but when I read `jenkins_host` I assume a hostname rather than a base url.

Reply:
Nope, that's all we get:
```json
 {"displayPath":"fdi-bootable-3.5.6.iso","fileName":"fdi-bootable-3.5.6.iso","relativePath":"result/fdi-bootable-3.5.6.iso"}
```

and `jenkins_host` is defined *with* schema ;)